public/static/js/tour-controller.js

```diff
@@ -106,12 +106,22 @@ dlangTourApp.controller('DlangTourAppCtrl', [ '$scope', '$http', 'hotkeys', func
 	hotkeys.add({
 		combo: 'left',
 		description: 'Go to previous section',
-		callback: prevPage
+		callback: function() {
```

Comment:
That doesn't look right. There was a PR which introduced those methods and now they are removed?